src/415D90.c

```diff
@@ -2054,29 +2054,23 @@ void func_802A5738(s32 arg0, s32 x, s32 y) {
     }
 }
 
-// regalloc, arg setting order dumbness
-#ifdef NON_EQUIVALENT
-void func_802A57C8(s32 arg0, s32 x, s32 y) {
-    s32 a0;
-
-    s32 x2;
-    s32 y2;
-
-    x2 = x + 11;
-    y2 = y + 6;
-
-    if (D_802AD610 == 0) {
-        a0 = 0x1D0000;
-        a0 |= 0xCB;
-    } else {
-        a0 = 0x1D0000;
-        a0 |= 0xCC;
+void func_802A57C8(s32* userData, s32 baseX, s32 baseY, s32 width, s32 height, s32 opacity, s32 darkening) {
+    s32 x = baseX + 11;
+    s32 y = baseY + 6;
+    s32 msgID;
```

Comment:
So happy you figured this one out - stumped us forever. Could you add a blank line after msgid decl?